KubeArmor/main.go

```diff
@@ -31,8 +36,10 @@ func init() {
 
 func main() {
 	if os.Geteuid() != 0 {
-		kg.Printf("Need to have root privileges to run %s\n", os.Args[0])
-		return
+		if os.Getenv("KUBEARMOR_UBI") == "" {
+			kg.Printf("Need to have root privileges to run %s\n", os.Args[0])
+			return
+		}
```

Comment:
We should not do this, KubeArmor should work with just caps and not root privileged in any image or scenario.

Reply:
We should move to caps and non root user in standard kubearmor image as well

Reply:
can we create a separate issue for it and handle there?

Reply:
Ref #1401